README.md

```diff
@@ -397,7 +400,7 @@ You might also be interested in checking out [awesome-shell](https://github.com/
 
  > Find better alternatives to the software you already use or a replacement for software you cannot or do not want to use.
 
-#### [Linux Foundation](www.linuxfoundation.org)
+#### [Linux Foundation](http://linuxfoundation.org)
```

Comment:
Could you replace this link with the full current one (`https://www.linuxfoundation.org/`)?